test/jasmine/tests/hover_label_test.js

```diff
@@ -1383,14 +1420,14 @@ describe('hover updates', function() {
                 Plotly.restyle(gd, 'marker.color', [colors0.slice()]);
             });
 
-            return assertLabelsCorrect([351, 251], [358, 272], '2');
+            return assertLabelsCorrect([351, 251], [358, 272], '2', 'events 0');
         })
         .then(unhover)
         .then(function() {
             expect(hoverCnt).toEqual(1);
             expect(unHoverCnt).toEqual(1);
 
-            return assertLabelsCorrect([400, 200], [435, 198], '3');
+            return assertLabelsCorrect([420, 100], [435, 198], '3', 'events 1');
```

Comment:
Ha. Found one. Why did this change?

Reply:
`x=400` is a little too far away using the new `compare` logic. I don't know exactly where the center of the marker is (that's why I've moved to more explicitly-sized-and-ranged plots for tests like this, so we can easily reason about the pixel positions of different points), probably the test would have worked just increasing to `405` or something but I just moved the mouse to a point I *knew* would be in range.